content/en/database_monitoring/setup_postgres/troubleshooting.md

```diff
@@ -189,7 +189,15 @@ See the section on [truncated query samples](#query-samples-are-truncated) for i
 
 #### Postgres extended query protocol
 
-If a client is using the Postgres [extended query protocol][9] or prepared statements, the Datadog Agent is unable to collect explain plans due to the separation of the parsed query and raw bind parameters. If the client provides an option to force using the simple query protocol, then turning that on enables the Datadog Agent to collect execution plans.
+If a client is using the Postgres [extended query protocol][9] or prepared statements on Postgres version 12 or newer, enable the following beta feature in the [Postgres integration config](https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/data/conf.yaml.example).
```

Comment:
Could you update this to use the reference link format, as mentioned in our contributing guidelines?
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#text

Reply:
https://github.com/DataDog/documentation/pull/16285/commits/57a78b5915a5b3de946c0bcfd5432340cd25c4a8